### PR TITLE
Fine RC Announcement

### PR DESCRIPTION
**Pull request by JPrause**

### CODE REVIEW

**Comment by JPrause** [comment]
@cybette we got the all-clear from QE on smoke testing. Can you review and merge.

**Comment by JPrause** [comment]
@cybette let me check on one last thing that popped up. Hold off on merging for now.

**Comment by JPrause** [comment]
Okay, issue checked and we're good-to-go.
@cybette please merge.

File: site/_posts/2017-05-04-Announcing-Fine-RC-Release.html.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Fine Beta 2 Release
```
Comment (cybette) [inline site/_posts/2017-05-04-Announcing-Fine-RC-Release.html.md]: This should say release candidate?

**Comment by JPrause** [comment]
@cybette thanks for catching that, update made.

**Pull request merged**